lib/web/apps.go

```diff
@@ -55,9 +55,9 @@ func (h *Handler) siteAppsGet(w http.ResponseWriter, r *http.Request, p httprout
 	return makeResponse(ui.MakeApps(h.auth.clusterName, h.proxyDNSName(), appClusterName, appServers))
 }
 
-type CreateAppSessionRequest struct {
-	// FQDN is the fully qualified domain name of the application.
-	FQDN string `json:"fqdn"`
+type AppResolveParams struct {
```

Comment:
Let's rename this to `AppResolveRequest` for consistency.

Reply:
I'm not sure about this since these `AppResolveParams` are populated in the course of both `getAppFQDN` and `createAppSession`. Renaming this to `AppResolveRequest` would misleadingly indicate this is just the input part of a single request-response pair.

Reply:
Fair enough, please make wrapper types for individual endpoints, like:
```go
type GetAppFQDNRequest AppResolveRequest
```
(or ember a struct, whichever is cleaner)